app/controllers/admin/content_controller.rb

```diff
@@ -37,6 +37,24 @@ def edit
     new_or_edit
   end
 
+  def reverse
+    @article = Article.find(params[:id])
```

Comment:
This `find` method will return an error if it cannot find an Article with this ID so I don't think it would get to the next line. Consider using an alternate method to find.